docs/docs/guides/web3_tree_shaking_support_guide/index.md

```diff
@@ -5,7 +5,7 @@ sidebar_label: Web3.tree.shaking
 
```

Comment:
The label of tree shaking guide should be "Web3 tree shaking guide" instead of `Web3.tree.shaking`

Reply:
in `sidebar_label: Web3.tree.shaking`